### PR TITLE
Added the Actions Icons on Cluster Detail Page

### PR DESCRIPTION
**Pull request by rhamitarora**
In Admin Panel, add 4 action icons on cluster detail page.

https://issues.redhat.com/browse/ARO-3488

### What this PR does / why we need it:

To improve user experience.

### CODE REVIEW

File: portal/v2/src/App.tsx [hunk 1]
```diff
@@ -328,9 +328,10 @@ function App(props: { params: any }) {
               params={props.params}
             />
           </Stack.Item>
-          <Stack.Item grow>
+          <Stack.Item grow>
```
Comment (anshulvermapatel) [inline portal/v2/src/App.tsx:331]: nit: these extra spaces can be removed.

File: portal/v2/src/ClusterDetail.tsx [hunk 2]
```diff
@@ -281,6 +300,30 @@ export function ClusterDetailPanel(props: {
...
+                }
+              />
+            </TooltipHost>
+            <TooltipHost content={`SSH`}>
+              <IconButton
+                iconProps={{ iconName: "CommandPrompt" }}
+                aria-label="SSH"
+                onClick={() => _onSSHClick(props.currentCluster)}
+              />
+            </TooltipHost>
```
Comment (anshulvermapatel) [inline portal/v2/src/ClusterDetail.tsx:325]: You can create a separate component for this and use that in here and in the `ClusterListComponent`.

File: portal/v2/src/App.tsx (hunk 1 above)
Comment (rhamitarora) [reply portal/v2/src/App.tsx]: Done

File: portal/v2/src/ClusterDetail.tsx (hunk 2 above)
Comment (rhamitarora) [reply portal/v2/src/ClusterDetail.tsx]: New "ToolIcons" component created and used in both Cluster Listing and Detail Page

File: portal/v2/src/ClusterDetail.tsx [hunk 3]
```diff
@@ -262,6 +267,20 @@ export function ClusterDetailPanel(props: {
     })
   }
 
+  const _onCopyResourceID = (item: any) => {
+    navigator.clipboard.writeText(item.resourceId)
+  }
+
+  const _onSSHClick = (item: any) => {
+    _dismissPanel()
```
[the remark this answers is not in the document]
Comment (rhamitarora) [reply portal/v2/src/ClusterDetail.tsx]: _dismissPanel removed and new Popup component added for SSH model

**Comment by anshulvermapatel** [comment]
Commendable work Amit. It is working great. Just wanted to point out few thing.
- The SSH popup should automatically close when the ClusterDetailPanel is closed. Currently it is still there even if we close the ClusterDetailPanel in the background.
- Please add e2e tests for this.

**Comment by rhamitarora** [comment]
> Commendable work Amit. It is working great. Just wanted to point out few thing.
> 
> * The SSH popup should automatically close when the ClusterDetailPanel is closed. Currently it is still there even if we close the ClusterDetailPanel in the background.
> * Please add e2e tests for this.

Changes done.

**Comment by anshulvermapatel** [comment]
@rhamitarora Can you please rebase.

File: portal/v2/src/ClusterDetail.tsx [hunk 4]
```diff
@@ -91,7 +93,7 @@ export function ClusterDetailPanel(props: {
   const [detailPanelVisible, setdetailPanelVisible] = useState<string>("Overview")
   const [customPanelStyle, setcustomPanelStyle] = useState<Partial<IPanelStyles>>({
     root: { top: "40px", left: "225px" },
-    content: { paddingLeft: 30, paddingRight: 5 },
+    content: { paddingLeft: 30, paddingRight: 5 },
```
Comment (anshulvermapatel) [inline portal/v2/src/ClusterDetail.tsx:96]: nit: extra white spaces can be removed.

File: portal/v2/src/ClusterDetail.tsx [hunk 5]
```diff
@@ -183,11 +185,12 @@ export function ClusterDetailPanel(props: {
...
   const _dismissPanel = () => {
     dismissPanel()
-    props.onClose() // useEffect?
+    props.sshBox.current.hidePopup()
+    props.onClose()
     setData([])
     setFetching("")
     setDataLoaded(false)
-    setError(null)
+    setError(null)
```
Comment (anshulvermapatel) [inline portal/v2/src/ClusterDetail.tsx:193]: nit: extra white spaces can be removed.

Comment (rhamitarora) [reply portal/v2/src/ClusterDetail.tsx]: done

File: portal/v2/src/ClusterDetail.tsx (hunk 4 above)
Comment (rhamitarora) [reply portal/v2/src/ClusterDetail.tsx]: done

**Review by Shivkumar13 — APPROVED** [review]

**Review by anshulvermapatel — APPROVED** [review]

**Pull request merged**